consensus/spos/worker.go

```diff
@@ -783,22 +784,20 @@ func (wrk *Worker) processEquivalentMessageUnprotected(cnsMsg *consensus.Message
 		return err
 	}
 
-	// TODO[Sorin next PR]: update EquivalentMessageInfo structure to hold also the proof(bitmap+signature) that was received
-	// then on commit block store this data on blockchain in order to use it on the next block creation
 	equivalentMsgInfo.Validated = true
+	equivalentMsgInfo.PreviousPubkeysBitmap = cnsMsg.PubKeysBitmap
```

Comment:
This should be the current pubkeys bitmap and the current aggregated signature, the previous one is already on the header